libr/bin/format/elf/elf_write.c

```diff
@@ -141,6 +141,10 @@ ut64 Elf_(r_bin_elf_resize_section)(struct Elf_(r_bin_elf_obj_t) *bin, const cha
 				phdrp->p_paddr += delta;
 			}
 		}
+		else if( phdrp->p_offset + phdrp->p_filesz >= rsz_offset + rsz_osize ){
```

Comment:
Please check https://github.com/radare/radare2/blob/master/DEVELOPERS.md for the coding style. We usually do:
```
if (..) {
<tab>...
} else if (...) {
<tab>...
}
```

Reply:
Sorry about that.
Changed my despicable else if.